### ISSUE TITLE
TranslationQueryset chokes on select_related(None)

### ISSUE THREAD

**Issue by spectras** [KristianOellegaard/django-hvad#189]
Django 1.6 introduced the ability to pass `None` to `select_related` to clear the list.
Right now, this is not supported by hvad. It tries so `.split('__')` it.

Have to check, but with the work to make `language()` lazy, it is quite possible moving most of the meat of `select_related` to query time as well would allow supporting Django semantics. The `select_related` method nees some love anyway. Great work has been done for its initial implementation and it's a shame it's still limited to 1-level depth.

Basically I believe `TranslationQueryset` should behave like django's queryset does: simple store settings and only work when needed.
